commands/config.js

```diff
@@ -12,23 +12,106 @@ exports.command = 'config';
 
 exports.describe = 'Configuration.';
 
-exports.builder = {
-  format: {
-    choices: ['table', 'yaml', 'json'],
-    description: 'What output format would you like?',
-    default: 'table'
-  },
-  out: {
-    description: 'Output to a file'
-  },
-  flatten: {
-    description: 'Flatten the config object into dot notation',
-    type: 'boolean',
-    default: false
-  }
+exports.builder = (yargs) => {
+  const electronPrebuiltVersion = require('electron-prebuilt/package.json').version;
+  const pkg = require('../lib/package');
+  var opts = {
+    verbose: {
+      describe: 'Confused or trying to track down a bug and want lots of debug output?',
+      type: 'boolean',
+      default: false
+    },
+    platform: {
+      describe: 'What platform are we building for?',
+      choices: ['win32', 'linux', 'darwin'],
+      default: process.platform
+    },
+    arch: {
+      describe: 'What platform architecture are we building for?',
+      choices: ['x64', 'x86'],
+      default: process.arch
+    },
+    electron_version: {
+      describe: 'What version of electron are we using?',
+      default: electronPrebuiltVersion
+    },
+    version: {
+      describe: 'What version of the application are we building?',
+      default: process.env.npm_package_version || pkg.version
+    },
+    name: {
+      describe: 'What is the kebab cased name of the application?',
+      default: process.env.npm_package_name || pkg.name
+    },
+    product_name: {
+      describe: 'What is the name of the application we should display to humans?',
+      default: pkg.productName || pkg.name
+    },
+    description: {
+      describe: 'What is the description of the application we should display to humans?',
+      default: process.env.npm_package_description || pkg.description
+    },
+    sign: {
+      describe: 'Should this build be signed?',
+      type: 'boolean',
+      default: true
+    },
+    signtool_params: {
```

Comment:
Is this only for electron-winstaller, or does it apply to electron-installer-codesign and/or electron-osx-sign?